tests/testthat/test_wrapper.R

```diff
@@ -40,6 +40,10 @@ test_that("Basic hcore functionality works", {
   expect_silent(run(hc, 2100))
   expect_silent(run(hc))
 
+  # All years should appear in the output
+  out <- fetchvars(hc, dates = 1745:2300)
+  expect_equivalent(sort(unique(out$year)), 1745:2300)
```

Comment:
Why are there two expect_silent(run) in a row after each other? Is the second run(hc) call just to run the date out to the year 2300 for all years should appear in the output? test?

Reply:
This is older code, not something I added. The model is at 2100 at that point, and those lines test that it will run to 2100 (no change) and then continue to 2300, all without error.